addons/purchase_requisition/models/purchase_requisition.py

```diff
@@ -0,0 +1,292 @@
+        """
+        res = self.env.ref('purchase.purchase_rfq')
+        res['context'] = self.env.context
+        res['domain'] = [('id', 'in', self[0].purchase_ids.ids)]
+        return res
+
+    def _prepare_purchase_order(self, supplier):
+        return {
+            'origin': self.name,
+            'date_order': self.date_end or fields.datetime.now(),
+            'partner_id': supplier.id,
+            'currency_id': self.company_id and self.company_id.currency_id.id,
+            'company_id': self.company_id.id,
+            'fiscal_position_id': self.env['account.fiscal.position'].get_fiscal_position(supplier.id),
+            'requisition_id': self.id,
+            'notes': self.description,
+            'picking_type_id': self.picking_type_id.id,
+        }
+
+    def _prepare_purchase_order_line(self, requisition_line, purchase_id, supplier):
+        product = requisition_line.product_id
+        default_uom_po_id = product.uom_po_id.id
+        ctx = self.env.context.copy()
+        ctx['tz'] = self.user_id.tz
+        date_order = self.ordering_date or fields.Datetime.now()
+        qty = self.env['product.uom']._compute_qty(requisition_line.product_uom_id.id, requisition_line.product_qty, default_uom_po_id)
+        taxes = product.supplier_taxes_id
+        fpos = supplier.property_account_position_id.id
+        taxes_id = fpos.map_tax(taxes) if fpos else []
+        res = self.env['purchase.order.line']._get_name_price_quantity_date(
+            requisition_line.product_id,
+            supplier,
+            date_order and date_order[:10],
+            qty,
+            product.uom_po_id,
+            purchase_id.currency_id,
+            order_id=purchase_id)
+
+        vals = {
+            'name': res['name'],
+            'order_id': purchase_id.id,
+            'product_qty': res['quantity'],
+            'product_id': product.id,
+            'product_uom': default_uom_po_id,
+            'price_unit': res['price_unit'],
+            'date_planned': res['date_planned'],
+            'taxes_id': [(6, 0, taxes_id)],
+            'account_analytic_id': requisition_line.account_analytic_id.id,
+        }
+
+        return vals
+
+    @api.multi
+    def make_purchase_order(self, partner_id):
+        """
+        Create New RFQ for Vendor
+        """
+        assert partner_id, 'Vendor should be specified'
+        res_partner = self.env['res.partner']
+        supplier = res_partner.browse(partner_id)
+        res = {}
+        for requisition in self:
+            if not requisition.multiple_rfq_per_supplier and supplier.id in filter(lambda x: x, [rfq.state != 'cancel' and rfq.partner_id.id or None for rfq in requisition.purchase_ids]):
+                raise UserError(_('You have already one %s purchase order for this partner, you must cancel this purchase order to create a new quotation.') % rfq.state)
+            ctx = self.env.context.copy()
+            ctx.update({'mail_create_nolog': True})
```

Comment:
Use ctx = dict(self.env.context, mail_create_nolog=True) instead of above two statements.